frame/dvm/src/tests.rs

```diff
@@ -43,7 +42,7 @@ fn default_erc20_creation_unsigned_transaction() -> UnsignedTransaction {
 		gas_limit: U256::from(0x100000),
 		action: ethereum::TransactionAction::Create,
 		value: U256::zero(),
-		input: FromHex::from_hex(ERC20_CONTRACT_BYTECODE).unwrap(),
+		input: array_bytes::hex2bytes_unchecked(ERC20_CONTRACT_BYTECODE),
```

Comment:
What will happen if pass a wrong erc20 contract bytecode?

Reply:
Good question. But ERC20_CONTRACT_BYTECODE is const here.

Reply:
This is in a test, I did not see.